locales/nl-NL/src/structure.ts

```diff
@@ -171,6 +171,9 @@ export default removeUndefinedLocaleResources({
   'confirm-delete-dialog.cdr-table.document-id.label': 'Document-ID',
   /** The toast title when the copy button has been clicked */
   'confirm-delete-dialog.cdr-table.id-copied-toast.title': 'Document-ID gekopieerd naar klembord!',
+  /** The toast title when the copy button has been clicked but copying failed */
+  'confirm-delete-dialog.cdr-table.id-copied-toast.title-failed':
+    'Kopiëren van document-ID mislukt',
```

Comment:
```suggestion
    'Kopiëren van document ID mislukt',
```

https://github.com/sanity-io/sanity/blob/master/packages/sanity/src/structure/i18n/resources.ts#L185